ArduCopter/Log.cpp

```diff
@@ -593,8 +546,6 @@ const struct LogStructure Copter::log_structure[] = {
       "PTUN", "QBfHHH",          "TimeUS,Param,TunVal,CtrlIn,TunLo,TunHi", "s-----", "F-----" },
     { LOG_OPTFLOW_MSG, sizeof(log_Optflow),
       "OF",   "QBffff",   "TimeUS,Qual,flowX,flowY,bodyX,bodyY", "s-EEEE", "F-0000" },
-    { LOG_NAV_TUNING_MSG, sizeof(log_Nav_Tuning),
```

Comment:
We should remove LOG_NAV_TUNING_MSG from the defines.h file

Reply:
removed thanks!